### PR TITLE
Issue220 relationship

### PR DESCRIPTION
**Pull request by LCCarmody**
Trying to edit the pharmacotherapy branch. THis is what I used for my logical definition

" "'pharmacotherapy' and ('therapeutic_uses' some 'chemical entity' and ('has_role' some %s))"
  vars: - drug_role"

But when running the reasoner, this puts 'pharmacotherapy' under 'independent continuant', which is not correct.

Not sure how to think about

### CODE REVIEW

**Comment by LCCarmody** [comment]
@matentzn no hurry, but this is the current state of things.

File: src/patterns/dosdp-patterns/pharmacotherapy_chemicalsubstance.yaml [hunk 1]
```diff
@@ -67,6 +70,6 @@ exact_synonym:
   value: syns
   
 equivalentTo:
-  text: "'pharmacotherapy' and ('inheres_in' some %s)"
+  text: "'pharmacotherapy' and ('therapeutic_uses' some 'chemical entity' and ('has_role' some %s))"
```
Comment (matentzn) [inline src/patterns/dosdp-patterns/pharmacotherapy_chemicalsubstance.yaml:73]: Should be:

```
text: "'pharmacotherapy' and ('therapeutic_uses' some ('chemical entity' and ('has_role' some %s)))"
```

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/patterns/dosdp-patterns/pharmacotherapy_chemicalsubstance.yaml (hunk 1 above)
Comment (LCCarmody) [reply src/patterns/dosdp-patterns/pharmacotherapy_chemicalsubstance.yaml:73]: works

**Comment by LCCarmody** [comment]
For some reason I cannot accept your change, but I did it manually, so I know it works. Will implement

**Comment by LCCarmody** [comment]
close #220

**Pull request merged**